### PR TITLE
extracted extensions

### PR DESCRIPTION
**Pull request by dmcclory**
@nicholaides & @csojinb-picwell - care to take a look?

### CODE REVIEW

File: schematics_extensions/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from schematics.types import (
+    IntType, StringType, FloatType, BooleanType, LongType)
```
Comment (csojinb-picwell) [inline schematics_extensions/__init__.py]: Why do we need to import all of these types here?

Comment (dmcclory) [inline schematics_extensions/__init__.py]: good call, we don't! just leftover from the extraction.

File: schematics_extensions/mockable_list_type.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from schematics.types.compound import ListType as SchematicsListType
+from random import randint
+
+
+class MockableListType(SchematicsListType):
+    def _mock(self, context=None):
+        return [self.model_class.get_mock_object()
```
Comment (csojinb-picwell) [inline schematics_extensions/mockable_list_type.py]: So, the bug that we found before is that this assumes that the ListType has a ModelType in it, instead of one of the basic types. (i.e. the ListType has no `model_class` attribute if it's a basic thing) Can we address that too? (Maybe in a separate PR?)

Comment (dmcclory) [inline schematics_extensions/mockable_list_type.py]: let's do a separate PR

**Comment by nicholaides** [comment]
:+1: Looks like it's in good-enough shape to start using.
